examples/workflows/query_workflow.rb

```diff
@@ -0,0 +1,22 @@
+class QueryWorkflow < Temporal::Workflow
+  def execute
+    state = "waiting for cancel"
+    last_signal_received = nil
+    workflow.on_query("state") do |input|
```

Comment:
nit: let's use consistent terminology here — either `input` or `args` (as in the GPRC interface)